src/components/AgreementSections/LinesList.js

```diff
@@ -84,9 +84,8 @@ export default class LinesList extends React.Component {
 
   renderPOLine = (line) => {
     const { orderLines } = this.props.agreement;
-
     if (!line.poLineId) return '';
-    if (!orderLines) return <Spinner />;
+    if (!orderLines.length) return <Spinner />;
```

Comment:
`orderLines` can be undefined based on `AgreementViewRoute:190`. It specifically _doesn't_ have a default `[]` value so that we can snoop whether the fetch has returned yet.

So I think this can crash, can't it?